package.json

```diff
@@ -29,10 +29,10 @@
     "@aws-cdk/aws-apigatewayv2-integrations-alpha": "^2.110.0-alpha.0",
     "@aws-cdk/aws-lambda-python-alpha": "2.110.0-alpha.0",
     "aws-cdk-lib": "^2.126.0",
+    "cargo-lambda-cdk": "^0.0.19",
```

Comment:
I'm wondering if this should go in a `package.json` inside the filemanager directory? I noticed @williamputraintan did this for the postgres manager.

Reply:
But then the pipeline/ environment no longer have access or install to this dependency as part of the CDK? The postgres manager has a `package.json` but it is for the runtime running the lambda

Reply:
Okay, sounds good. One approach could be to have multiple yarn workspaces as well? Although I think for now that's probably unnecessary for one dependency.

Reply:
Yup, yup. Well spotted, Marko. We're gonna hit the `yarn workspaces` in tick, real soon. Let sit tight for the moment, pls.